docs/guides/debugging.md

```diff
@@ -338,7 +338,7 @@ Move the coredump and the unpackaged relocatable package into some dir
 
 ```
 (host)$ cd /path/to/scylla # with the right commit checked out
-(host)$ ./tools/toolchain/dbuild -it -v $dir:/workdir:z -- bash -l
+(host)$ ./tools/toolchain/dbuild -it -v $dir:/workdir -- bash -l
```

Comment:
Somehow I missed this line in my review (I guess I reviewed an earlier version without this line).
What do you mean "doesn't need to be relabled anymore"? What changed? Isn't this a docker vs. podman difference rather than something that changed?

Reply:
Possibly. All I know is that I needed this `:z` with F34 but don't anymore with F35. Dropping `:z` also allows me to run this without `sudo` too. I never bothered to investigate why exactly this `:z` was needed in the first place.

Reply:
Note that dbuild itself still uses ":z" on all the directories it mounts with "-v", so in your command line you'd be mounting several directories with relabeling (:z) and one extra one without.
I guess whether or not the ":z" matters depends on your selinux configuration, so maybe something in it changed for you during the upgrade?

Reply:
In F34 I used to have to disable selinux (`sudo setenforce 0`) and launch the container with `sudo`. With F35 it started working without either of these out-of-the-box. Note that the directory that I usually attach `/home/bdenes/ScyllaDB` is a large directory tree, containing everything related to my work. It is possible (actually almost certain) that it contains files not owned by me (owned by `root`) and that this was the reason I had to use `sudo` (otherwise these couldn't be relabeled).